doc/mgr/dashboard.rst

```diff
@@ -69,29 +72,36 @@ aspects of your Ceph cluster:
 * **Cluster logs**: Display the latest updates to the cluster's event and
   audit log files. Log entries can be filtered by priority, date or keyword.
 * **Hosts**: Display a list of all hosts associated to the cluster, which
-  services are running and which version of Ceph is installed.
+  disk are attached, which services are running and which version of Ceph is
+  installed.
 * **Performance counters**: Display detailed service-specific statistics for
   each running service.
 * **Monitors**: List all MONs, their quorum status, open sessions.
-* **Monitoring**: Enables creation, re-creation, editing and expiration of
-  Prometheus' Silences, lists the alerting configuration of Prometheus and
-  currently firing alerts. Also shows notifications for firing alerts. Needs
-  configuration.
+* **Monitoring**: Enable creation, re-creation, editing and expiration of
+  Prometheus' Silences, list the alerting configuration of Prometheus and all
+  configured and firing alerts. Show notifications for firing alerts.
 * **Configuration Editor**: Display all available configuration options,
   their description, type and default values and edit the current values.
 * **Pools**: List all Ceph pools and their details (e.g. applications,
-  placement groups, replication size, EC profile, CRUSH ruleset, etc.)
+  autoscaling, placement groups, replication size, EC profile, CRUSH rulesets,
```

Comment:
I'll prefer `pg-autoscaling`, it's more precise.
```suggestion
  pg-autoscaling, placement groups, replication size, EC profile, CRUSH rulesets,
```

Reply:
Good suggestion, amended accordingly.